src/prefect_server/services/graphql/server.py

```diff
@@ -51,9 +51,24 @@ def health(request: Request) -> JSONResponse:
     return JSONResponse(dict(status="ok", version=app_version))
 
 
+def _get_uvicorn_log_level():
+    prefect_log_level = prefect_server.configuration.config["logging"]["level"]
+    # Uvicorn log levels are lower case
+    uvicorn_log_level = prefect_log_level.lower()
+    if uvicorn_log_level in uvicorn.config.LOG_LEVELS:
+        logger.info(f"Using uvicorn log level = {repr(uvicorn_log_level)}")
```

Comment:
You can do quick reprs in Python string formatting
```suggestion
        logger.info(f"Using uvicorn log level = {uvicorn_log_level!r}")
```